test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -149,15 +149,15 @@ def test_postgres_dbt_commands_with_cwd_as_project_dir(self):
 
     @use_profile('postgres')
     def test_postgres_dbt_commands_with_randomdir_as_project_dir(self):
-        workdir = os.getcwd()
+        workdir = self.test_root_dir
```

Comment:
Without this change, integration tests failed on MacOS because of how the os handles temp dir/file. Functionally, `os.getcwd()` and `self.test_root_dir` are the same value. `os.getcwd()` returns a fully resolved filepath while `self.test_root_dir` is a string value pointing to a symlinked path that is equivalent to `os.getcwd()`.